index.html

```diff
@@ -44,6 +44,7 @@
 <li><a href="{{ site.url }}users/grechanp/">Pavel Grechanuk</a> (<a href="https://github.com/grechasneak" class="user-mention">@grechasneak</a>)</li>
 <li><a href="{{ site.url }}users/schicklr/">Robert Schickler</a> (<a href="https://github.com/albeanth" class="user-mention">@robertschickler</a>)</li>
 <li><a href="{{ site.url }}users/tamashia/">Aaron Tamashiro</a> (<a href="https://github.com/aarontama" class="user-mention">@aarontama</a>)</li>
+<li><a href="{{ site.url }}users/stewryan/">Ryan Stewart</a> (<a href="https://github.com/ryanstwrt" class="user-mention">@ryanstwrt</a>)</li>
```

Comment:
Change the order to last name alpha please

Reply:
it took me longer than I care to admit to realize you meant to make it alphabetically haha

Reply:
Alrighty, I updated on my branch. Thanks for catching some of the spelling errors, I appreciate it. Do I need to create another pull request for this most recent update?

Reply:
@ryanstwrt you're good man! We can see all of your most recent changes.